python/python/tests/test_async_client.py

```diff
@@ -3271,6 +3271,28 @@ async def test_time(self, redis_client: TRedisClient):
         assert int(result[0]) > current_time
         assert 0 < int(result[1]) < 1000000
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_lastsave(self, redis_client: TRedisClient):
+        yesterday = date.today() - timedelta(1)
+        yesterday_unix_time = time.mktime(yesterday.timetuple())
+
+        result = await redis_client.lastsave()
+        assert isinstance(result, int)
+        assert result > yesterday_unix_time
```

Comment:
This assumes that the last time the database was saved was yesterday?  Maybe include a note about this so if this does fail, someone just needs to re-run after running BGSAVE().